### ISSUE TITLE
Can I generate single report collecting the xml reports already generated?

### ISSUE THREAD

**Issue by jpgpuyo** [Kotlin/kotlinx-kover#645]
Hello,
First of all, thanks for your effort, you are creating an amazing tool :)

Right now, I'm working in a big multimodule project and I am executing unit tests in multiple executions.

So, I would like to know if, once are executions are completed, is it possible to generate single report without executing unit tests again?

Thanks!

**Comment by shanshin** [comment]
Hi,
could you clarify, are you running multiple executions on the same machine? 
if yes, then the Gradle build cache is suitable for you, so the tests will not be reruned, and the coverage value will be taken from the cache.

**Comment by jpgpuyo** [comment]
Thanks for your quick response @shanshin 
I have 2 use cases:
- Execute the tests in parallel with github actions. In this case, still trying to find the best solution, as I don't think that I can execute jobs in parallel in the same runner.
- Execute tests locally in my laptop. In this case, maybe I can use gradle cache.

For solve the issue with github actions, I was thinking about upload all the xml to github artifacts, and then create another task to download all xmls and generate coverage, but not sure if in this case I will take advantage of gradle cache. I will investigate.

If no more solutions can be provided, then you can close this question :)

**Comment by shanshin** [comment]
This is a solvable issue, but the solution is not very convenient: there are special tasks called `koverBinaryReport` or `koverBinaryReport<Variant>`. 

You can generate an HTML or XML report from several binary reports, but for this you need to pass the paths to all sources and all compiled classes.  This can be done using the [Kover CLI](https://kotlin.github.io/kotlinx-kover/cli/#generating-reports) or a special Java dependency: `org.jetbrains.kotlinx:kover-features-jvm:0.8.1` and [function](https://github.com/Kotlin/kotlinx-kover/blob/06f4de485860009869f00504f4ad8b569871dbdc/kover-features-jvm/src/main/java/kotlinx/kover/features/jvm/KoverLegacyFeatures.kt#L85)

**Comment by shanshin** [comment]
Closed as answered.
Feel free to create new issue if you have any additional questions.

**Comment by shanshin** [comment]
The ability to use coverage from already generated reports has been added in `0.9.0-RC`

**Comment by jpgpuyo** [comment]
Thanks a lot for notifying me @shanshin 
Could you provide more information about how to use this new feature?
Thanks!
